.github/workflows/buildtest.yaml

```
@@ -42,4 +42,8 @@ jobs:
         run: npm ci
 
       - name: Run Build Command
-        run: npm run build
\ No newline at end of file
+        run: npm run build
+        env:
+          # AI Services
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          TAVILY_API_KEY: ${{ secrets.TAVILY_API_KEY }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)